src/control/server/instance_storage_rpc.go

```diff
@@ -332,7 +332,7 @@ func scanEngineBdevsOverDrpc(ctx context.Context, engine Engine, pbReq *ctlpb.Sc
 				RdbSize:  pbReq.RdbSize,
 			}
 			upd, err := populateCtrlrHealth(ctx, engine, bhReq, c,
-				hwprov.DefaultPCIeLinkStatsProvider())
+				pciutils.NewPCIeLinkStatsProvider())
```

Comment:
Not unless it adds some value, IMO. `hwprov.DefaultPCIeLinkStatsProvider()` was just a wrapper around `return pciutils.NewPCIeLinkStatsProvider()`. The other constructors exist to prevent redundant code being used to set up complex providers with the same configuration in different parts of the codebase.